### DISCUSSION TITLE
What happened to Ton mining?

### DISCUSSION THREAD

**Discussion by GodSlayer01** [Lolliedieb/lolMiner-releases#1632] [General]
What happened to Ton mining? I saw that the difficulty went up like 1000x so now it isn't profitable at all.
Is it the only way to mine it now to do it by mining it alone?

**Comment by DmitryKoichev** [comment]
Yea, I see that from last withdraw 4 days have been passed and my 2 of 3090 farmed only 0.66 ton, but usually it was ~1 ton per day. Today just updated 1.52 release in HiveOs and ton has been stopped. Could anyone please explain what happened ?

**Reply by DmitryKoichev** [reply]
@GodSlayer01 
Ok Looks like I found the reason:
https://ton.app/endmining

POS instead of POW - that's all. 2 days remaining for mining TON

**Reply by GodSlayer01** [reply]
This is what I thought but it doesn't explain the sudden drop in rewards and increase in difficulty. A huge pool (infiniteton.com) with more than 2 PH/s came out of nowhere and ruined our last days to mine TON. Now you can get about the same amount of TON as before if you use this pool but you can't dual mine, it has to be only TON mining. I think you will be able to mine even after the PoW period ends if you use this pool. It looks like they'll use your GPUs to mine whatever is more profitable or rent your equipment and then pay in TON, similar to the way Nicehash does it with BTC.
https://miningpoolstats.stream/toncoin
https://infiniteton.com/

**Reply by DmitryKoichev** [reply]
Thanks for the info, but I switched to pure ETH while POW not ended yet
